crates/sui-framework/sources/governance/validator_cap.move

```diff
@@ -5,7 +5,7 @@ module sui::validator_cap {
     use sui::object::{Self, ID, UID};
     use sui::transfer;
     use sui::tx_context::TxContext;
-    friend sui::sui_system;
+    friend sui::sui_system_state_inner;
```

Comment:
If we want to update to `SuiSystemStateInnerV2` that's in a different module `sui_system_state_inner_v2`, I suppose it is fine to add a `friend sui::sui_system_state_inner_v2;` here? @amnn